restful.module

```diff
@@ -472,15 +481,21 @@ function restful_menu_process_callback($resource_name, $version = NULL) {
     $handler->setHttpHeaders('Access-Control-Allow-Origin', $allowed_origin);
   }
 
-  $path = implode('/', $path);
-
   $method = strtoupper($_SERVER['REQUEST_METHOD']);
 
   if ($method == \RestfulInterface::POST && \RestfulManager::getRequestHttpHeader('X-HTTP-Method-Override')) {
-    $method = strtoupper(\RestfulManager::getRequestHttpHeader('X-HTTP-Method-Override'));
+    $method = \RestfulManager::getRequestHttpHeader('X-HTTP-Method-Override');
```

Comment:
Why are you deleting `strtoupper`? There is nothing that ensures that the value of the `X-HTTP-Method-Override` HTTP header will be uppercase.